react/features/base/translation/ConfigLanguageDetector.js

```diff
@@ -0,0 +1,25 @@
+/* global config */
+
+/**
+ * Custom language detection, just returns the config property if any.
+ */
+export default {
+    /**
+     * Name of the language detector.
+     */
+    name: 'configLanguageDetector',
+
+    /**
+     * The actual lookup.
+     *
+     * @returns {string} The default language if any.
+     */
+    lookup() {
+        return config.defaultLanguage;
+    },
+
+    /**
+     * Doesn't support caching.
+     */
+    cacheUserLanguage: Function.prototype
```

Comment:
is this supposed to be a noop function? In that case, how about `() => {}`, I find it easier to understand.